v1/app/static/css/style.css

```diff
@@ -15,7 +15,7 @@
 
 .board .btn-success {
     margin-top: 10px;
-    margin-bottom: 10px;
+    margin-bottom: 50px;
```

Comment:
Pls change the front-end and not the v1. It's only as a backup.

Reply:
@abishekvashok done similarly in the front-end folder.

Reply:
Pls remove the changes done in v1,
Sorry if I wasn't crystal clear last time :) @rupav